src/model_instance.rs

```diff
@@ -535,6 +548,22 @@ impl ModelInstance {
             }
         }
 
+        if cmd_arguments.is_present("ffm_mc_iteration_count") {
+            if let Some(val) = cmd_arguments.value_of("ffm_mc_iteration_count") {
+                let hvalue = val.parse::<>()?;
+                mi.ffm_mc_iteration_count = hvalue;
```

Comment:
What exactly is the meaning of the two new hpos?

Reply:
`ffm_mc_iteration_count` - number of iteration that the monte carlo will be run
`ffm_mc_dropout_rate` - dropout rate for it
Or did I misunderstand?

Reply:
Ok cool, just wanted to make sure I understood :) Intuitively `mc_iteration_count=1` + `mc_dropout_rate=0.0` imply current mode of operation then -- this is not entirely true anymore though right?

Reply:
current = current main